docs/api/connections.rst

```diff
@@ -26,8 +26,10 @@ Compatibility
 Connection / Contact / Joints   Rod  Rigid Body
 =============================== ==== =========== 
 FreeJoint                       ✅   ❌
+GenericSystemTypeFreeJoint      ✅   ✅
 ExternalContact                 ✅   ❌
 FixedJoint                      ✅   ❌
+GenericSystemTypeFixedJoint     ✅   ✅
```

Comment:
please remove from docs, we don't host experimental features here yet.

Reply:
Ok, I see. I removed it in https://github.com/GazzolaLab/PyElastica/pull/149/commits/63be1a41d9aac93826a52d3171766d4984ab2d45